api/src/api/notes.js

```diff
@@ -43,6 +43,13 @@ router.post(
   '/',
   requireLead,
   errorWrap(async (req, res) => {
+    req.body.metaData.versionHistory.push({
+      default: [],
+      date: Date.now(),
+      action: Note.actions.CREATED,
+      memberID: req.user._id,
+    });
+
     const note = await Note.create(req.body);
     res.status(200).json({
       success: true,
```

Comment:
Add a message here and move the note to a `data` tag like in the `put` endpoint.